core/standalone/build.gradle

```diff
@@ -162,6 +162,10 @@ dependencies {
     testCompile 'org.scalatest:scalatest_2.12:3.0.5'
 }
 
+tasks.withType(ScalaCompile) {
```

Comment:
Better to move this to `build.gradle` within `subprojects` part such that it always gets applied on all project

```
pluginManager.withPlugin('scala') {
        tasks.withType(ScalaCompile) {
            scalaCompileOptions.additionalParameters = gradle.scala.compileFlags
        }
    }
```

Reply:
I do have that on my TODO list to do in a separate PR. I punted on it in this one as it'll touch all scala projects and I wanted to do some more consolidation in that. Shall I throw it all onto this PR still?

Reply:
Then lets do that in separate PR